positionContract.sol

```diff
@@ -0,0 +1,255 @@
+pragma solidity ^0.5.2;
+import {CErc20, ComptrollerInterface, CToken} from "./CERC20.sol";
+import {CEther} from "./CEther.sol";
+import {UniswapExchangeInterface} from "./uniswap.sol";
+import {ERC20Interface} from "./ERC20.sol";
+
+
+contract positionContract {
+    // The user's address
+    address public ownerAddress; 
+    string public tradeType; 
```

Comment:
if you wanted to be extra awesome here, I suggest having a single enum

ZRX_SHORT
ZRX_LONG
REP_SHORT
REP_LONG

etc...

Which will save on storage.
Or more generally, you could use a mask here.

IDK how important it is to write gas efficient code for these things. It's gonna be a while before all the money saved from these small gas optimizations amount to a parity wallet hack 😂